doc/sphinx/source/introduction.rst

```diff
@@ -102,16 +102,18 @@ Core development team
 ---------------------
 
 -  Deutsches Zentrum für Luft- und Raumfahrt (DLR), Institut für Physik
-   der Atmosphäre, Germany (PI)
+   der Atmosphäre, Germany (Co-PI)
 
-   - ESMValTool Core PI and Developer: contact for requests to use the ESMValTool and for collaboration with the development team, access to the PRIVATE GitHub repository.
+   - ESMValTool Core Co-PI and Developer: contact for requests to use the ESMValTool and for collaboration with the development team, access to the PRIVATE GitHub repository.
```

Comment:
Maybe it would be a bit more clear if this sentence was not nested in the list of participating institutions? Adding the email addresses mentioned in the NOTICE file would also make it more clear.